tinygrad/codegen/linearizer.py

```diff
@@ -263,24 +265,6 @@ def render_loop(xx:List[Variable]) -> Tuple[UOp, ...]:
       # define accumulator
       acc = self.global_load(0, global_idxs+local_idxs+fake_reduce_idxs+upcast_idxs, self.get_reduce_acc(self.reduceop))
 
-      if (tc:=self.tensor_core):
-        def calc_tc_idxs(local_size: int, aliases: List[List[int]]):
-          replace_idxs = []
-          for alias in aliases:
-            full_var, full_var_sz = NumNode(0), 1
-            if alias[0] != 0:
-              for i in alias:
-                next_var = local_idxs[-i] if i > 0 else Variable("_uidx_tc", 0, local_size-1)
-                full_var += next_var * full_var_sz
-                full_var_sz *= next_var.max+1
-            replace_idxs.append(full_var)
-          return replace_idxs
-        replace_acc_idxs = calc_tc_idxs(tc.thread_local_sizes[2], tc.thread_local_aliases[2])
```

Comment:
IIRC dropping the replace_acc_idxs logic is gonna make future support of WMMA and Opt.GROUP/GROUPTOP harder in the future, but since it's explicitly rejected by apply_opts, I think it's well worth dropping it now and simplifying the thread aliases.  And it's not even clear that there are perf advantages to that combo.